monorepo/packages/wallet-connectors/src/NearConnector/near.ts

```diff
@@ -61,58 +69,6 @@ function getNearConfig(env = 'testnet') {
   }
 }
 
-export const near = new NearConnector({
-  normalizeChainId: false,
-  normalizeAccount: false,
-  config: getNearConfig('testnet'),
-});
```

Comment:
what is the reason to move this to `src/NearConnector/index.ts` ?

Reply:
same of above